packages/esm-patient-list-app/src/ui-components/overflow-menu.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useCallback, useEffect, useRef } from 'react';
+
+interface CustomOverflowMenuComponentProps {
+  menuTitle: React.ReactNode;
+}
+
+const CustomOverflowMenuComponent: React.FC<CustomOverflowMenuComponentProps> = ({ menuTitle, children }) => {
+  const [showMenu, setShowMenu] = useState(false);
+  const wrapperRef = useRef(null);
+  const toggleShowMenu = useCallback(() => setShowMenu((state) => !state), []);
```

Comment:
I think that you don't need `useCallback` for this one (though @brandones might be better to evaluate that).

Reply:
Yeah I agree; `useCallback` is probably counterproductive here. If you're really curious you can always use the [profiler](https://reactjs.org/blog/2018/09/10/introducing-the-react-profiler.html) to find out.

Reply:
Good thing to know how to use, but certainly not important here :) I'd just delete the `useCallback`.

As an aside... my dream is that one day someone in the MFE community will elect themselves "the perf audit person," helping identify the slow parts of the code across the application.